Doc/library/pdb.rst

```diff
@@ -639,6 +639,54 @@ can be overridden by the local file.
 
    Print the return value for the last return of the current function.
 
+.. pdbcommand:: exceptions [excnumber]
+
+   List or jump between chained exceptions.
+
+   When using ``pdb.pm()``  or ``Pdb.post_mortem(...)`` with a chained exception
+   instead of a traceback, it will now allow the user to move between the
+   chained exceptions using ``exceptions`` command to list exceptions, and
+   ``exception <number>`` to switch to that exception.
```

Comment:
```suggestion
   ``exception <number>`` to switch to that exception. In the case of ``ExceptionGroup``s this will also allow to inspect sub exceptions.
```

Reply:
I don't think this feature, as is, should allow traversing into exception groups. The arg implies a linear order of exceptions, and if you have an exception group in an exception chain, and each exception in the group can also be in a chain, it's not obvious how to flatten all this into a linear order.

Reply:
Ok, removed.